src/plugins/vis_builder/public/visualizations/table/components/table_viz_options.tsx

```diff
@@ -0,0 +1,227 @@
+          label={
+            <>
+              {i18n.translate('visTypeTableNewNew.params.perPageLabel', {
+                defaultMessage: 'Max rows per page',
+              })}
+              ,
+              <EuiIconTip
+                content={
+                  <FormattedMessage
+                    id="visTypeTableNewNews.field.emptyTooltip"
+                    defaultMessage="Leaving this field empty means it will use number of buckets from the response."
+                  />
+                }
+                position="right"
+              />
+            </>
+          }
+          isInvalid={!isPerPageValid}
+          min={1}
+          paramName="perPage"
+          value={styleState.perPage}
+          setValue={(_, value) =>
+            setOption((draft) => {
+              draft.perPage = value;
+            })
+          }
+        />
+
+        <SwitchOption
+          label={i18n.translate('visTypeTableNewNew.params.showMetricsLabel', {
+            defaultMessage: 'Show metrics for every bucket/level',
+          })}
+          paramName="showMetricsAtAllLevels"
+          value={styleState.showMetricsAtAllLevels}
+          setValue={(_, value) =>
+            setOption((draft) => {
+              draft.showMetricsAtAllLevels = value;
+            })
+          }
+          data-test-subj="showMetricsAtAllLevels"
+        />
+
+        <SwitchOption
+          label={i18n.translate('visTypeTableNewNew.params.showPartialRowsLabel', {
+            defaultMessage: 'Show partial rows',
+          })}
+          tooltip={i18n.translate('visTypeTableNewNew.params.showPartialRowsTip', {
+            defaultMessage:
+              'Show rows that have partial data. This will still calculate metrics for every bucket/level, even if they are not displayed.',
+          })}
+          paramName="showPartialRows"
+          value={styleState.showPartialRows}
+          setValue={(_, value) =>
+            setOption((draft) => {
+              draft.showPartialRows = value;
+            })
+          }
+          data-test-subj="showPartialRows"
+        />
+
+        <SwitchOption
+          label={i18n.translate('visTypeTableNewNew.params.showTotalLabel', {
+            defaultMessage: 'Show total',
+          })}
+          paramName="showTotal"
+          value={styleState.showTotal}
+          setValue={(_, value) =>
+            setOption((draft) => {
+              draft.showTotal = value;
+            })
+          }
+        />
+
+        <SelectOption
+          label={i18n.translate('visTypeTableNewNew.params.totalFunctionLabel', {
+            defaultMessage: 'Total function',
+          })}
+          disabled={!styleState.showTotal}
+          options={totalAggregations}
+          paramName="totalFunc"
+          value={styleState.totalFunc}
+          setValue={(_, value) =>
+            setOption((draft) => {
+              draft.totalFunc = value;
+            })
+          }
+        />
```

Comment:
I think this should really be just a boolean of whether to show it or not, without a selectable total function. The total row should reflect the metric for each column as if there were no split rows, not use the same aggregation function on all metrics. For example, a column of averages should show the average across all rows (and not simply sum the row averages or average them).

Reply:
Hey @joshuarrrr, I think this could be an enhancement for the table vis. Currently, since we just want to refactor the table vis, should we just keep everything same for customer?

Reply:
Currently, it is a selectable list because customer could choose the metric as the total function. 
<img width="226" alt="Screen Shot 2022-11-03 at 00 28 56" src="https://user-images.githubusercontent.com/79961084/199667253-de5b70ca-f54f-4cc7-a4e0-c3cabbc83ad0.png">

So I don't get it why it should be a boolean, though the name `total function` is a bit confusing. If you think the current total function feature is not good, we could open a feature enhancement issue. I will discuss with you.

Reply:
Yeah, the existing feature is just completely wrong and broken. The idea of a "total function" to apply to all columns is a bug (statistical not coding), not an enhancement,  because it generates silly meaningless numbers even when implemented "correctly".